datasrc/skins/greensward.json

```diff
@@ -0,0 +1,29 @@
+{"skin": {
```

Comment:
is this a new json style standard?

Reply:
Ye seems like...
also the }} at the end.

Reply:
side node: also "value":"true" instead of "value":true

Reply:
@AssassinTee yes sure I fully agree also with @Ryozuki. But this is nothing that belongs in this pr. Consistent style is better than mixed style. We could open a new pr refactoring all skins and discuss it there if we even want to do that.